### PR TITLE
Set dev Procfile making it easy to run in dev mode

### PR DESCRIPTION
**Pull request by ratneshraval**
## JIRA
n/a

## Description
- Procfile to run webpack and rails with single command `yarn dev`
- You will need to install gem `foreman`

## Tests

- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I ran all my tests locally which were green.
- [ ] My code adds no new issues to Code Climate
- [ ] I ran all the linters for the project.
- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [ ] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: Procfile.dev [hunk 1]
```diff
@@ -0,0 +1,2 @@
+web: bundle exec rails s
+hot: ./bin/webpack-dev-server
```
Comment (wmitchell-cambria) [inline Procfile.dev]: I'm not an expert with Foreman. Do you think this should use `webpack-dashboard` like the `yarn start` command does? (I don't like the dashboard, so I just run `./bin/webpack-dev-server` directly.)

Comment (ratneshraval) [reply Procfile.dev]: This will run both webpack (watch mode) and rails.

**Comment by denys-davydov** [comment]
@ratneshraval Is a developer supposed to install **foreman** manually or you can add it to the development group in **Gemfile**?

**Comment by ratneshraval** [comment]
@denys-davydov  both are possible options. In cals we have it declared in `gemfile`. I think that will be easier for everyone. I'll update.
@wmitchell-cambria I'll update readme.

**Comment by ratneshraval** [comment]
Knock knock...

**Comment by wmitchell-cambria** [comment]
The code looks safe to me, but since I am biased towards having fewer different ways to run the application, I'm going to wait for another developer to approve it. That way we have at least two people who would actually use it (because I would not).

**Comment by ratneshraval** [comment]
retest this please

**Pull request merged**